projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -68,16 +82,41 @@ export class NavigationListComponent implements OnChanges {
   @Output()
   public readonly collapsedChange: EventEmitter<boolean> = new EventEmitter();
 
+  public usePageLevelTimeRange: boolean = false;
+
   public activeItem$?: Observable<NavItemLinkConfig | undefined>;
 
+  public navItems$?: Observable<NavItemConfig[]>;
+
   public constructor(
     private readonly navigationService: NavigationService,
     private readonly activatedRoute: ActivatedRoute,
-    private readonly navListComponentService: NavigationListComponentService
+    private readonly navListComponentService: NavigationListComponentService,
+    private readonly timeRangeService: TimeRangeService,
+    private readonly featureStateResolver: FeatureStateResolver
   ) {}
 
   public ngOnChanges(): void {
     this.navItems = this.navListComponentService.resolveFeaturesAndUpdateVisibilityForNavItems(this.navItems);
+
+    this.navItems$ = this.featureStateResolver
+      .getCombinedFeatureState([ApplicationFeature.PageTimeRange, ApplicationFeature.NavigationRedesign])
+      .pipe(
+        map(featureState => featureState === FeatureState.Enabled),
+        tap(usePageLevelTimeRange => {
+          this.usePageLevelTimeRange = usePageLevelTimeRange;
+
+          return usePageLevelTimeRange;
```

Comment:
this return isn't required since the value from the previous pipe section will flow into the next one.

Reply:
done